src/main/resources/Help/Code/Translations.xml

```diff
@@ -0,0 +1,251 @@
+help.macros.useMacro.title=Use Macros
+help.macros.viewMoreMacros=View more macros
+help.macros.warningMacro=Warning Macro
+help.mainXwikiFeatures.description=Congratulations for acquiring your XWiki instance. XWiki is very versatile and can be used to create knowledge bases, intranets or any type of business application you want. This is achievable because of XWiki's customization power, that starts at page level, from a simple to a more advanced usage.
+help.mainXwikiFeatures.title=Main XWiki features
+help.pageEditing.description=Use a powerful {0}WYSIWYG{1} content editor or {2}syntax{3} to create rich wiki pages.
+help.pageEditing.title=Page Editing
+help.seeExamples.label=See examples
+help.templates.article.description=Article page
+help.templates.article.myNewArticle=My new article
+help.templates.article.title=Article
+help.templates.contents=Contents
+help.templates.references=References
+help.templates.paragraph=Paragraph
+help.templates.simple=Simple
+help.templates.subParagraph=Sub-paragraph
+help.templates.subSubParagraph=Sub-sub paragraph
+help.templates.createYourTemplate.description=Integrate structure and macros in an existing page, and transform it in a template.
+help.templates.createYourTemplate.tip={0}Template Providers{1}  expose templates on the Create Page dialog so that the users can use them.
+help.templates.createYourTemplate.title=Create your template
+help.templates.description=Templates are predefined pages that are copied when creating a new page.
+help.templates.encyclopedia.credits=Credits
+help.templates.encyclopedia.description=Encyclopedia page
+help.templates.encyclopedia.falabella=Falabella
+help.templates.encyclopedia.image=Image
+help.templates.encyclopedia.miniatureHorse=Miniature horse
+help.templates.encyclopedia.morphology=Morphology
+help.templates.encyclopedia.price=Price
+help.templates.encyclopedia.region=Region
+help.templates.encyclopedia.title=Encyclopedia
+help.templates.examples.description=Want to start from a predefined template? Here are some examples of available templates inside XWiki
+help.templates.examples.title=Examples
+help.templates.figure=Figure
+help.templates.meeting.actionItems=Action Items
+help.templates.meeting.agenda=Agenda
+help.templates.meeting.attendees=Attendees
+help.templates.meeting.conferenceRoom=CONFERENCE ROOM
+help.templates.meeting.date=04 SEPTEMBER 20XX
+help.templates.meeting.hour=10:00 AM
+help.templates.meeting.lastMeetingFollowUp=Last Meeting Follow-up
+help.templates.meeting.newBusiness=New Business
+help.templates.meeting.nextMeetingAgenda=Next Meeting Agenda
+help.templates.meeting.notes=Notes
+help.templates.meeting.summary=Summary
+help.templates.meeting.teamMeeting=Team Meeting
+help.templates.meetingReport.description=Agenda and notes
+help.templates.meetingReport.title=Meeting Report
+help.templates.sea=Sea
```

Comment:
I would prefer here the key to use ".image" instead of ".sea". What if we change the image to something else?